libcxx/test/tools/clang_tidy_checks/CMakeLists.txt

```diff
@@ -9,6 +9,17 @@ set(Clang_DIR_SAVE ${Clang_DIR})
 # versions must match. Otherwise there likely will be ODR-violations. This had
 # led to crashes and incorrect output of the clang-tidy based checks.
 find_package(Clang ${CMAKE_CXX_COMPILER_VERSION})
+if(NOT Clang_FOUND)
+  message(STATUS "Clang-tidy tests are disabled since the "
+                 "Clang development package is unavailable.")
+  return()
+endif()
+if(NOT TARGET clangTidy)
```

Comment:
Review note: This happens in the bootstrap build. I still think we should fix the bootstrap build; however we need to handle this case too.